ignite/contrib/metrics/regression/manhattan_distance.py

```diff
@@ -21,6 +21,16 @@ class ManhattanDistance(_BaseRegression):
 
     __ https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.DistanceMetric.html
 
+    Args:
```

Comment:
Maybe we should remove the `__init__` method ?

Reply:
we still need to do `super().__init__`, right?

Reply:
`_BaseRegression` does not implement `__init__` and inherits from `Metric`. If no `__init__` in this class, `__init__` from `Metric` is called. Here, this `__init__` is a useless proxy to the `__init__` from `Metric` class. What do you think ?

Reply:
Thanks for pointing out. Changed

Reply:
What about adding a comment on that point in the docstring ? I mean that the arguments relie on `Metric` `__init__`